ehitchhiking-ui/src/app/shared/components/trip/trip.component.ts

```diff
@@ -8,6 +10,7 @@ import {Trip} from '../trips-modal/trips';
 })
 export class TripComponent implements OnInit {
 	@Input() trip: Trip;
+	isRating: boolean;
```

Comment:
bad naming: isRating, pls rename to isRatingEditorVisible

Reply:
fixed